### ISSUE TITLE
Edit and import metadata

### ISSUE THREAD

**Issue by olivierlenard** [TabularEditor/TabularEditor#1078]
Hi, 
Great script the 'export' and 'import' Metadata especially when filling are creating many and many descriptions
What is the best way to edit the file after exporting it. I tried with excel, but all formating goes wrong adding " to formula ( for example, if a formula contains  x ="this text" it become x=**"**" this text "**"**

Any suggestion?!

**Comment by otykier** [comment]
Michael Kovalsky has an excellent blog post that describes how you can use `Microsoft.Office.Interop` to save/load files from Tabular Editor in .xlsx format, which should help avoid this problem. The scripts are located here: https://github.com/m-kovalsky/Tabular/tree/master/Descriptions

**Comment by olivierlenard** [comment]
hi @otykier ,
Actually, I went through Michael Kovalsky great (once again) blog post.
We are facing difficulties in importing the description once updated on excel. 
In parallel I kept on working on your script and simply deleted the "**Expression**" part on the first tsv (selectedobject). The "expressions" are not exported, and consequently not updated when importing the .tsv
My issue is not fixed, but does not exist anymore ☺
